apispec/ext/marshmallow/swagger.py

```diff
@@ -216,6 +216,23 @@ def field2length(field):
     return attributes
 
 
+def field2nullable(field, spec):
+    """Returns the dictionary with an appropriate nullable key depending on
+    the OpenAPI version.  Returns an empty dict if the field is not nullable
+
+    :param Field field: A marshmallow field.
+    :param spec: apispec.core.APISpec instance
+    :rtype: dict
+    """
+    if not field.allow_none:
+        return {}
+
+    if spec and spec.openapi_version.version[0] == 3:
```

Comment:
`spec.openapi_version.version[0]`

This is accessed in two places, and more to come. Maybe we should add a shortcut for it.
For instance properties `spec.major_version`, `spec.minor_version`.

Nothing blocking, just a thought.